app/cdap/components/WrangleHome/Components/WrangleCard/index.tsx

```diff
@@ -128,16 +86,15 @@ const WrangleCard = () => {
 
       SVG: ImportDatasetIcon,
     });
-
     setState({
```

Comment:
don't name this setState - functional hook components should have individual setStates in as much as possible so this should be setConnectorType. Also setState is an old reserved word in react so we should call it something else to avoid confusion.